fcs/samples/Tokenizer/Tokenizer.fsproj

```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.6.2" />
-    <PackageReference Include="FSharp.Core" Version="4.5.2" />
```

Comment:
Why is 4.5.2 being removed here?

Reply:
As you can see there are two package references.  Clearly one is redundant.

Reply:
As for bumping the fsharp.core version 4.6.2 has already shipped, this is in preparation for shipping the next one.  Which will likely happen when we ship a new compiler, in Dev 16.1

Reply:
4.5.2 removal makes sense here, but is it necessary (from a build process) to rev 4.6.2 with a new compiler if we're not shipping changes?

Reply:
@cartermp, this is in preparation for shipping the next FSharp.Core package.  Whether we ship it on a particular date or note is a decision for then.